### PR TITLE
Clarify that a primary key is required #7936

### PR DESCRIPTION
**Pull request by mehov**

### CODE REVIEW

**Comment by mehov** [comment]
#7936

File: en/orm/associations.rst [hunk 1]
```diff
@@ -510,7 +510,7 @@ example above we would need tables for ``articles``, ``tags`` and
 ``articles_tags``.  The ``articles_tags`` table contains the data that links
 tags and articles together. The joining table is named after the two tables
 involved, separated with an underscore by convention. In its simplest form, this
-table consists of ``article_id`` and ``tag_id``.
+table consists of ``article_id`` and ``tag_id``, as well as `multiple-column ``PRIMARY KEY`` index <https://dev.mysql.com/doc/refman/8.4/en/create-table.html#idm44961976278224>`_ spanning both of these columns.
```
Comment (ADmad) [inline en/orm/associations.rst:513]: We can't assume that the reader is using MySQL.

Comment (dereuromark) [reply en/orm/associations.rst:513]: > In its simplest form, this table consists of ``article_id`` and ``tag_id`` as well as the default (primary key) ``id``.

would be more correct either way.

Any multi colum setups are not simple or trivial here IMO.

Comment (markstory) [reply en/orm/associations.rst:513]: I think the link is the problematic part as it is MySQL specific, and seems like a URL that will rot due to the fragment id.

**Pull request merged**